src/containers/SideBar/Miner/Miner.tsx

```diff
@@ -1,50 +1,51 @@
-import Scheduler from './components/Scheduler.tsx';
-import useAppStateStore from '../../../store/appStateStore.ts';
-import ModeSelect from './components/ModeSelect.tsx';
+import { useEffect, useState } from "react";
+import Tile from "./components/Tile.tsx";
+import { MinerWrapper, TileWrapper } from "./styles.ts";
+import AutoMiner from "./components/AutoMiner.tsx";
+import Scheduler from "./components/Scheduler.tsx";
+import useAppStateStore from "../../../store/appStateStore.ts";
+import ModeSelect from "./components/ModeSelect.tsx";
+import { formatNumber, truncateString } from "../../../utils";
 
 function Miner() {
-  const { cpuUsage, hashRate, cpuBrand, estimatedEarnings } = useAppStateStore((state) => ({
-    cpuUsage: state.cpuUsage,
-    hashRate: state.hashRate,
+  const [currResource, setCurrResource] = useState<"CPU" | "GPU">("CPU");
+  const { cpuUsage, hashRate, cpuBrand, gpuBrand, estimatedEarnings } =
+    useAppStateStore((state) => ({
+      cpuUsage: state.cpuUsage,
+      hashRate: state.hashRate,
       cpuBrand: state.cpuBrand,
-estimatedEarnings: state.estimatedEarnings
-  }));
-    const truncateString = (str: string, num: number): string => {
-        if (str.length <= num) {
-            return str;
-        }
-        return str.slice(0, num) + '...';
+      gpuBrand: state.gpuBrand,
+      estimatedEarnings: state.estimatedEarnings,
+    }));
+
+  useEffect(() => {
+    const resourceSwitcher = setInterval(() => {
+      setCurrResource((prev) => (prev === "CPU" ? "GPU" : "CPU"));
+    }, 5_000);
+
+    return () => {
+      clearInterval(resourceSwitcher);
     };
+  }, []);
 
-    function formatNumber(value: number): string {
-        if (value < 0) {
-            return value.toPrecision(1);
-        } else if (value >= 1_000_000) {
-            return (value / 1_000_000).toFixed(1).replace(/\.0$/, '') + 'm';
-        } else if (value >= 1_000) {
-            return (value / 1_000).toFixed(1).replace(/\.0$/, '') + 'k';
-        } else {
-            return value.toString();
-        }
-    }
+  const resourceBrand = currResource === "CPU" ? cpuBrand : gpuBrand;
 
   return (
-    <MinerContainer>
+    <MinerWrapper>
       <AutoMiner />
       <Scheduler />
-      <TileContainer>
+      <TileWrapper>
         <Tile title="Resources" stats="CPU" />
         <ModeSelect />
-        {/*<Tile title="GPU Utilization" stats="23%" />*/}
-        <Tile title="Hashrate (to remove)" stats={hashRate + ' H/s'} />
-        <Tile title="CPU Utilization" stats={cpuUsage + '%'} />
-        <Tile title="CHIP/GPU" stats={truncateString(cpuBrand, 10)} />
-        <Tile title="Est Earnings" stats={ formatNumber(estimatedEarnings / 1000000) + " XTM/24h"} />
-      </TileContainer>
-    </MinerContainer>
+        <Tile title="Hashrate (to remove)" stats={hashRate + " H/s"} />
+        <Tile title="CPU Utilization" stats={cpuUsage + "%"} />
+        <Tile title={currResource} stats={truncateString(resourceBrand, 30)} />
```

Comment:
```suggestion
        <Tile title={currResource} stats={truncateString(resourceBrand, 10)} />
```
Keep the formatting the same.